File: KnowledgeSpaceApk/adminapp/src/main/java/com/example/adminapp/HomeworkActAdm/HomeworkActAdm.java
package com.example.adminapp.HomeworkActAdm;

import androidx.appcompat.app.AppCompatActivity;
import androidx.fragment.app.FragmentManager;
import androidx.fragment.app.FragmentTransaction;

import android.os.Bundle;
import android.view.View;
import android.widget.Toast;

import com.example.adminapp.R;
import com.google.android.material.floatingactionbutton.FloatingActionButton;

public class HomeworkActAdm extends AppCompatActivity {

    private FloatingActionButton createHwBtn;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_homework_act_adm);
        FragmentManager fragmentManager = getSupportFragmentManager();
        FragmentTransaction fragmentTransaction = fragmentManager.beginTransaction();
        fragmentTransaction.replace(R.id.parentFrameLayoutHomeworkAdm,new HomeworkDefaultFragAdm());
        fragmentTransaction.commit();


    }
}

File: KnowledgeSpaceApk/adminapp/src/main/java/com/example/adminapp/HomeworkActAdm/HomeworkActAdm.kt
package com.example.adminapp.HomeworkActAdm

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.example.adminapp.R
import com.google.android.material.floatingactionbutton.FloatingActionButton

class HomeworkActAdm : AppCompatActivity() {
    private val createHwBtn: FloatingActionButton? = null
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_homework_act_adm)
        val fragmentManager = supportFragmentManager
        val fragmentTransaction = fragmentManager.beginTransaction()
        fragmentTransaction.replace(R.id.parentFrameLayoutHomeworkAdm, HomeworkDefaultFragAdm())
        fragmentTransaction.commit()
    }
}